content/applications/finance/payment_providers/stripe.rst

```diff
@@ -2,132 +2,135 @@
 Stripe
 ======
 
-`Stripe <https://stripe.com/>`_ is a United States-based online payment solution provider, allowing
+`Stripe <https://stripe.com/>`_ is a United States-based online payment solution provider allowing
 businesses to accept **credit cards** and other payment methods.
 
-Link your Stripe Account with Odoo
-==================================
+Create your Stripe account with Odoo
```

Comment:
If a user already has a Stripe account and they just want to _connect_ their existing Stripe account with Odoo, do they follow the same process? If so, I think it would be worth including that ("Create or connect your Stripe account with Odoo"). If not, another section that explains how to connect an existing account would be valuable.